docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -7,13 +7,24 @@
     "# Overview of User Defined Functions with cuDF"
```

Comment:
<p>In other parts of this notebook, we use the second person "you", whereas here we use "the user". Let's be consistent -- I have a strong preference for the second person "you".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/10607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>